client/src/i18n/en-US.json

```diff
@@ -765,7 +765,7 @@
         "share": "Share"
       },
       "placeholder": "Add a comment …",
-      "scroll_to_top": "Scroll to top"
+      "scroll_to_top": "Scroll to top."
```

Comment:
This also makes the mistake of using periods as punctuation for button labels.

![Screenshot of the Scroll to Top button using a period as punctuation](https://github.com/MESH-Research/Pilcrow/assets/2668987/c99291c2-3692-4789-a273-1018fdb470ac)

Reply:
See above.

Reply:
I removed the period for this.